.github/workflows/chromatic.yml

```diff
@@ -1,19 +1,10 @@
 name: Chromatic
 
-on:
-  workflow_run:
-    workflows: ['Build']
-    types:
-      - completed
-    branches:
-      - main
+on: push
 
 jobs:
-  chromatic:
-    name: Deploy chromatic
-    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+  tests:
     runs-on: ubuntu-latest
-    timeout-minutes: 10
```

Comment:
I would keep this setting because I saw some jobs gets stuck at some point and consume all the minutes remaining in the GitHub account for pipelines